Reference managed instances in installation docs (#18023)
sourcegraph/sourcegraph: doc/admin/install/managed.md

# Managed Sourcegraph instances

> Managed instances are provision and managed by the Sourcegraph team, making them a simple choice for customers to deploy Sourcegraph without having to worry about managing it.

Reviewer: Small typo—should be "provisioned" I think.

Author: ```suggestion
Managed instances are provisioned and managed by the Sourcegraph team, making them a simple choice for customers to deploy Sourcegraph without having to worry about managing it.
```

Sourcegraph provisions your instance in a completely isolated and secure cloud infrastructure. It will be restricted to only your organization through your enterprise VPN and/or SSO provider of choice.

## Costs

Please [contact us](https://about.sourcegraph.com/contact/sales) to discuss [pricing](https://about.sourcegraph.com/pricing) and to start a trial.

## Scaling

> Managed instances support most small to large enterprises well. Please [contact us](https://about.sourcegraph.com/contact/sales) to discuss options.

Reviewer: Do we maybe want to change this to "Managed instances can be used by most enterprises of any size. Please contact us..."? We don't specify what "small" or "large" means in this context and since it's basically trying to suggest "we can probably handle you" I think it might be useful to alter the framing.

Author: ```suggestion
Managed instances can be used by most enterprises of any size. Please [contact us](https://about.sourcegraph.com/contact/sales) to discuss options.
```

I like that. We have not built the support for really massive ones yet, but if there is appetite this is something we should build.
